lib/itamae/logger.rb

```diff
@@ -48,10 +48,25 @@ def color(code, &block)
       %w!debug info warn error fatal unknown!.each do |level|
         module_eval(<<-EOC, __FILE__, __LINE__ + 1)
           def #{level}(msg)
-            super("  " * indent_depth + msg)
+            super(indent_msg(msg))
```

Comment:
Can we also achieve this by `"  " * indent_depth + super(msg)`? I want to delegate work of `#indent_msg` to `Formatter#msg2str`.